.github/workflows/notification.yml

```diff
@@ -0,0 +1,95 @@
+name: GSI Chatroom PR Notification
+
+on:
+  pull_request:
+    types: [opened, reopened]
+  push:
+    branches:
+      - main
+    
+
+jobs:
+  notify:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Pull Request Details
+      run: |
+        echo "Pull Request: ${{ github.event.pull_request.title }}"
+        echo "Author: ${{ github.event.pull_request.user.login }}"
+
+    - name: Google Chat Notification
+      run: |
+        curl --location --request POST '${{ secrets.WEBHOOK_URL }}' \
```

Comment:
Where are the secrets stored?

Reply:
They are stored on the [repo environment](https://docs.github.com/en/actions/security-guides/encrypted-secrets).